script.js

```diff
@@ -61,25 +60,26 @@ colorApp.getColors = (userChoice) => {
             } 
         })
         .then((jsonData) => {
-            colorApp.displayColors(jsonData);
+            console.log(jsonData)
+            // colorApp.displayColors(jsonData);
```

Comment:
```suggestion
colorApp.displayColors(jsonData);
```
UNcommented-out this, since we will update displayColors to work with the new API :)